src/base/os.lua

```diff
@@ -11,7 +11,6 @@
 --
 
 	premake.override(os, "execute", function(base, cmd)
-		cmd = path.normalize(cmd)

```

Comment:
This line caused some weird issues. I don't understand, why it is here.
I have tryed to run simple command to copy files:
```lua
os.execute("{COPY} ../../Code/Dependencies/FBX/lib/".._ACTION.."/x64/debug/libfbxsdk.dll ../../Bin64")
```
But this command is put through path.normalize and it's output is:
**"{COPY} ../../Code/Dependencies/FBX/lib/vs2013/x64/Bin64"**